chart/README.md

```diff
@@ -0,0 +1,134 @@
+# Gitpod Self-Hosted
+
+[gitpod.io](https://gitpod.io) is a servide that provides developers with ready-to-code dev environments in the cloud.
+
+This [Helm](https://helm.sh) chart allows you to deploy and operate an instance of Gitpod on your own infrastructure.
+
+
+## Prerequisites
+
+- Kubernetes 1.13+
+- Helm 3+
+
+
+## Get Repo Info
+
+```console
+helm repo add charts.gitpod.io https://charts.gitpod.io
+helm repo add stable https://charts.helm.sh/stable
+helm repo add stable https://helm.min.io/
+helm repo update
+```
+
+_See [helm repo](https://helm.sh/docs/helm/helm_repo/) for command documentation._
+
+
+## Install Chart
+
+```console
+# Helm
+$ helm install $(for i in $(cat configuration.txt); do echo -e "-f $i"; done) gitpod .
+```
+
+_See [configuration](#configuration) below._
+
+_See [helm install](https://helm.sh/docs/helm/helm_install/) for command documentation._
+
+
+## Dependencies
+
+By default this chart installs additional, dependent charts:
+
+- [stable/docker-registry](https://github.com/helm/charts/tree/master/stable/docker-registry)
+- [stable/minio](https://github.com/minio/charts)
+- [stable/mysql](https://github.com/helm/charts/tree/master/stable/mysql)
+
+_See [configuration](#configuration) for options to replace those dependencies._
+
+_See [helm dependency](https://helm.sh/docs/helm/helm_dependency/) for command documentation._
+
+
+## Uninstall Chart
+
+```console
+# Helm
+$ helm uninstall [RELEASE_NAME]
```

Comment:
In the install example, the release name is `gitpod`, right? Shouldn't we use `gitpod` as the release name here as well instead of `[RELEASE_NAME]`?